### PR TITLE
Minor cleanup as per PR suggestions

### PR DESCRIPTION
**Pull request by abhimanyugupta07**

### CODE REVIEW

**Comment by abhimanyugupta07** [comment]
Good to hear from you @massdosage :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 NOTE: This CHANGELOG tracks changes in the 5.x (Hive 2.x) line. For 6.x (Hive 3.x) and above please refer to https://github.com/klarna/HiveRunner/blob/main/CHANGELOG.md.
 
+## [5.5.0] - 2023-01-23
+### Changed
+- Upgraded to log4j2.
```
Comment (massdosage) [inline CHANGELOG.md:10]: This already happened in the 5.4.1 release which unfortunately we can't undo so you either need to "yank" that one and do this, or just move this comment down there. My vote would go for a yank and a 5.5.0 release.

Comment (abhimanyugupta07) [reply CHANGELOG.md:12]: done !

File: pom.xml [hunk 2]
```diff
@@ -45,10 +45,11 @@
   </properties>
   
   <dependencies>
+   <!--Hive uses Guava 14.0.1-->
```
Comment (abhimanyugupta07) [inline pom.xml:48]:
```suggestion
   <!--Hive 2.3.9 uses Guava 14.0.1-->
```

**Review by massdosage — APPROVED** [review]
LGTM, who did the release yesterday? Ideally we'd do `5.5.0` today so it matches the CHANGELOG date, otherwise we can update that and do it tomorrow?

**Comment by abhimanyugupta07** [comment]
I will start a release build.

**Comment by massdosage** [comment]
OK cool, I forget who has permissions to do what. Thanks for tidying this last bit up :)

**Pull request merged**